<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistogramView</class>
 <widget class="QWidget" name="HistogramView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>381</width>
     <height>751</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QmitkDataStorageComboBox" name="DataSelector"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="PlotBtn">
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QwtPlot" name="HistogramPlot" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="StartPointBtn">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="StartPointLE"/>
      </item>
      <item>
       <widget class="QPushButton" name="EndPointBtn">
        <property name="text">
         <string>End</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EndPointLE"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="ProfilePlotBtn">
        <property name="text">
         <string>Profile Plot</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QwtPlot" name="ProfileHistogramPlot" native="true"/>
    </item>
    <item>
     <widget class="QLabel" name="ImageWidget">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QWidget</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header>QmitkDataStorageComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
